File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/SwapButtonFragment.java
package com.regula.facesample.items.customization.fragment;

import android.view.View;
import android.widget.ImageButton;

import androidx.annotation.NonNull;

import com.regula.facesample.R;
import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 7.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class SwapButtonFragment extends FaceDefaultUiFragment {

    @Override
    protected View getSwapCameraButton(@NonNull View v) {
        View swapCameraBtn = super.getSwapCameraButton(v);
        if (swapCameraBtn instanceof ImageButton) {
            ((ImageButton) swapCameraBtn).setImageResource(R.drawable.swap_button);
        }
        return swapCameraBtn;
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/fragments/SwapButtonFragment.kt
package com.regula.facesamplekotlin.items.fragments

import android.view.View
import android.widget.ImageButton
import com.regula.facesamplekotlin.R
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class SwapButtonFragment : FaceDefaultUiFragment() {

    override fun getSwapCameraButton(v: View): View? {
        val swapCameraBtn = super.getSwapCameraButton(v)
        if (swapCameraBtn is ImageButton) {
            swapCameraBtn.setImageResource(R.drawable.swap_button)
        }
        return swapCameraBtn
    }
}